genesis/process/disabled/disabledBalanceComputationHandler.go

```diff
@@ -0,0 +1,40 @@
+package disabled
+
+import "math/big"
+
+// BalanceComputationHandler -
+type BalanceComputationHandler struct {
```

Comment:
maybe rename the file from disabledBalanceComputationHandler.go -> balanceComputationHandler.go

Reply:
done